script.js

```diff
@@ -0,0 +1,142 @@
+// Temperature Conversion Functions
+function farenheitDisplay(f) {
+    f = Number(tempInput.value).toFixed(2);
+    document.getElementById("farenheitValue").innerText = `${f} \xB0F`;
+}
+
+
+function fToC(f, c) {
+    f = Number(tempInput.value);
+    c = ((f - 32) * (5/9)).toFixed(2);
+    document.getElementById("celsiusValue").innerText = `${c} \xB0C`;
+}
+
+function fToK(f, k) {
+    f = Number(tempInput.value);
+    k = ((f + 459.67) * (5/9)).toFixed(2);
+    document.getElementById("kelvinValue").innerText = `${k} \xB0K`;
+}
+
+function fToR(f, r) {
+    f = Number(tempInput.value);
+    r = (f + 459.67).toFixed(2);
+    document.getElementById("rankineValue").innerText = `${r} \xB0R`;
+}
+
+function celsiusDisplay(c) {
+    c = Number(tempInput.value).toFixed(2);
+    document.getElementById("celsiusValue").innerText = `${c} \xB0C`;
+}
+
+function cToF(c, f) {
+    c = Number(tempInput.value);
+    f = (c * 1.8 + 32).toFixed(2);
+    document.getElementById("farenheitValue").innerText = `${f} \xB0F`;
+}
+
+function cToK(c, k) {
+    c = Number(tempInput.value);
+    k = (c + 273.15).toFixed(2);
+    document.getElementById("kelvinValue").innerText = `${k} \xB0K`;
+}
+
+function cToR(c, r) {
+    c = Number(tempInput.value);
+    r = ((c + 273.15) * 1.8).toFixed(2);
+    document.getElementById("rankineValue").innerText = `${r} \xB0R`;
+}
+
+function kelvinDisplay(k) {
+    k = Number(tempInput.value).toFixed(2);
+    document.getElementById("kelvinValue").innerText = `${k} \xB0K`;
+}
+
+function kToF(k, f) {
+    k = Number(tempInput.value);
+    f = (k * 1.8 - 459.67).toFixed(2);
+    document.getElementById("farenheitValue").innerText = `${f} \xB0F`;
+}
+
+function kToC(k, c) {
```

Comment:
You can get the same result with less conversion methods, if you had a method to convert each scale in Fahrenheit and from Fahrenheit into the other three, you would only have 6 equations